### ISSUE TITLE
Shuffling in dataloader

### ISSUE THREAD

**Issue by bart1** [point-cloud-radar/bird-cloud-gnn#132]
From everything I gather it is advised to shuffle data in the data loader so not each epoch gets offered data in the same order. 
e.g. : https://machinelearningmastery.com/training-a-pytorch-model-with-dataloader-and-dataset/ and https://www.leaky.ai/dataloader

**Comment by bart1** [comment]
/close
I see this is done by the sampler
